api/db-cdr/csv/achilles_analysis.csv

```diff
@@ -7,27 +7,24 @@ analysis_id,"analysis_name","stratum_1_name","stratum_2_name","stratum_3_name","
 5,"Number of persons by ethnicity","ethnicity_concept_id","","","","","column","counts"
 6,"Number of persons by age decile","age_decile","","","","","column","counts"
 10,"Number of all persons by year of birth by gender","year_of_birth","gender_concept_id","","","","column","counts"
-11,"Number of non-deceased persons by year of birth by gender","year_of_birth","gender_concept_id","","","","column","counts"
 12,"Number of persons by race and ethnicity","race_concept_id","ethnicity_concept_id","","","","pie","counts"
-603,"Number of distinct procedure occurrence concepts per person","","","","","","column","counts"
-606,"Age distribution by procedure_concept_id","procedure_concept_id","gender_concept_id","","","","box","distribution"
 1806,"Distribution of age by measurement_concept_id","measurement_concept_id","gender_concept_id","null","null","null","box","distribution"
+1814,"Distribution of numeric values by measurement concept id ","measurement_concept_id",NULL,NULL,NULL,NULL,"box","distribution"
 1815,"Distribution of numeric values, by measurement_concept_id and unit_concept_id","measurement_concept_id","unit_concept_id","unit_concept_name","","","box","distribution"
 1816,"Distribution of low range, by measurement_concept_id and unit_concept_id","","","","","","column","counts"
 1817,"Distribution of high range, by observation_concept_id and unit_concept_id","","","","","","column","counts"
-1900,"Measurement response gender distribution","measurement_concept_id","gender_concept_id","","value","","column","counts"
-1901,"Measurement response age distribution","measurement_concept_id","age_decile","","value","","column","counts"
-2000,"Number of patients with at least 1 Dx and 1 Rx","","","","","","column","counts"
-2001,"Number of patients with at least 1 Dx and 1 Proc","","","","","","column","counts"
-2002,"Number of patients with at least 1 Meas, 1 Dx and 1 Rx","","","","","","column","counts"
-2003,"Number of patients with at least 1 Visit","","","","","","column","counts"
+1900,"Measurement response gender histogram","measurement_concept_id","gender_concept_id","","value","","column","counts"
+1901,"Measurement response age at occurrence histogram","measurement_concept_id","age_decile","","value","","column","counts"
 3000,"Participant Count","concept_id","bin_size","domain_id","unit for measurements","","column","counts"
 3004,"Number of persons with at least one concept by concept_id by calendar year by gender by age decile","concept_id","calendar year","gender_concept_id","age decile","","column","counts"
-3006,"Age distribution by gender","concept_id","gender_concept_id","","","","box","distribution"
+3005,"Counts per person distribution by concept id","measurement_concept_id",NULL,NULL,NULL,NULL,"box","distribution"
```

Comment:
Oh those are the value distribution? 3005 Says Counts per person distribution -- not measurement value. I think that that will apply to all concepts. It will be interesting sometime to know The distribution of number of times a person has any generic concept like Diabetes or whatever.